service/http/response.go

```diff
@@ -31,16 +33,16 @@ func NewResponse(p *roadrunner.Payload) (*Response, error) {
 
 // Write writes response headers, status and body into ResponseWriter.
 func (r *Response) Write(w http.ResponseWriter) error {
+	p, h := handlePushHeaders(r.Headers)
```

Comment:
Willing to hear what you think about this. Headers are passed by reference and modified inside of the function (removing push header if present). I'm doing this for performance reasons, but I'm aware that this might not be desirable and can be refactored into copying original headers before passing them down this function. Same happens with `handleTrailers()`